com.unity.cinemachine/Editor/Upgrader/UpgradeObjectToCm3.cs

```diff
@@ -130,6 +93,68 @@ public GameObject UpgradeComponents(GameObject go)
             return notUpgradable;
         }
 
+        public void ProcessAnimationClip(AnimationClip animationClip, Animator trackAnimator)
```

Comment:
Everything else in this class supports Undo.  Would it be easy to add Undo support here?  If not, at least add a comment that this method has no Undo support.  I use Undo a lot when testing the upgrader.  It's really useful.

Reply:
I'll check